libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -647,8 +648,14 @@ class MapboxNavigation @VisibleForTesting internal constructor(
      */
     @Deprecated(message = "use a function withe the callback instead")
```

Comment:
```suggestion
    @Deprecated(message = "use a function with the callback instead")
```

nit: in case we're keeping it deprecated

Reply:
fixed :+1: